.github/workflows/ci.yaml

```diff
@@ -14,10 +14,5 @@ jobs:
         with:
           fetch-depth: 0
 
-      - name: set up golang
```

Comment:
The commit is missing the [KNI] prefix but yet the CI passed. I really need to revisit the script

Reply:
yup, this is what I was testing.